driver/pkcs11.mli

```diff
@@ -61,6 +61,17 @@ module Initialize_arg : sig
   val t :  t Ctypes.typ
 end
 
+module Nss_initialize_arg : sig
+  type _ck_nss_c_initialize_args
+  type t = _ck_nss_c_initialize_args Ctypes.structure
+  val flags : (CK_FLAGS.t, t) Ctypes.field
+  val t :  t Ctypes.typ
+
+  (* The unwrapped type is a string representing parameters to pass to the library *)
```

Comment:
I'm not sure I understand this comment.  The fact that this represents the parameters can be inferred from the name of the module.  Maybe the comment could say what well-defined format the string should be in?

Reply:
I've tried to make it clearer. In particular, the key thing is that the unwrapped type only allows you to set `LibraryParameters` - you can't provide any of the mutex functions for example.